foodRequest.py

```diff
@@ -1,26 +1,38 @@
 #!flask/bin/python
+import logging
+import sys
+
 from flask import Flask, jsonify, request
 from flask import abort
 from flask_cors import CORS, cross_origin
 app = Flask(__name__)
 cors = CORS(app)
 app.config['CORS_HEADERS'] = 'Content-Type'
 
-from Rojak import Rojak
-rojak = Rojak()
-
+from RojakCore import KnowledgeGraph
+rojak = KnowledgeGraph()
 app = Flask(__name__)
 
+# Logging Start
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(formatter)
+root.addHandler(handler)
+# Logging Ends
+
 @app.route('/todo/api/v1.0/recipes/', methods=['POST'])
 @cross_origin()
 def get_data():
     if "content" in request.json:
-        name = json.loads(request.json)["content"]
+        query = request.json["content"]
```

Comment:
Noob question, what's the difference between using and not using `json.loads()`?

Reply:
`request.json` returns `json` format we don't have to load it in `json` library again.